specs/validator/tableland-openapi-spec.yaml

```diff
@@ -0,0 +1,304 @@
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Tableland docs
+  url: http://docs.tableland.xyz
+servers:
+  - url: https://testnet.tableland.xyz/api/v1
+tags:
+  - name: query
+    description: Query the Tableland network
+    externalDocs:
+      description: Find out more about queries
+      url: http://docs.tableland.xyz
+  - name: receipt
+    description: Get information about transaction progress
+    externalDocs:
+      description: Find out more about receipts
+      url: http://docs.tableland.xyz
+  - name: tables
+    description: Access to table information
+    externalDocs:
+      description: Find out more about tables
+      url: http://docs.tableland.xyz
+paths:
+  /health:
+    get:
+      tags:
+        - health
+      summary: Returns OK if the validator considers it's healty.
+      description: Returns OK if the validator considers it's healthy.
+      operationId: health
+      responses:
+        "200":
+          description: The validator is healthy.
+  /version:
+    get:
+      tags:
+        - version
+      summary: Returns version information about the validator daemon.
+      description: Returns version information about the validator.
+      operationId: version
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/VersionInfo"
+  /query:
+    get:
+      tags:
+        - query
+      summary: Query the Tableland network
+      description: Returns the results of a SQL read query against the Tabeland network
+      operationId: queryFromQuery
+      parameters:
+        - name: statement
+          in: query
+          description: The SQL read query statement
+          required: true
+          schema:
+            type: string
+        - name: format
+          in: query
+          description: >
+            The requested response format:
+             * `objects` - Returns the query results as a JSON array of JSON objects.
+             * `table` - Return the query results as a JSON object with columns and rows properties.
+          required: false
+          schema:
+            type: string
+            enum:
+              - objects
+              - table
+            default: "objects"
+        - name: extract
+          in: query
+          description: Whether or not to extract the JSON object from the single property of the surrounding JSON object.
+          required: false
+          schema:
+            type: boolean
+            default: false
+        - name: unwrap
+          in: query
+          description: Whether or not to unwrap the returned JSON objects from their surrounding array.
+          required: false
+          schema:
+            type: boolean
+            default: false
+      responses:
+        "200":
+          description: successful operation
```

Comment:
Tiny nit: Mix of lower (successful) and Sentence (Invalid) case in descriptions.